contracts/TalentLayerID.sol

```diff
@@ -104,7 +104,9 @@ contract TalentLayerID is ERC2771RecipientUpgradeable, ERC721Upgradeable, UUPSUp
      * @dev Returns the total number of tokens in existence.
      */
     function totalSupply() public view returns (uint256) {
-        return nextProfileId.current() - 1;
+        unchecked {
```

Comment:
I don't think it worth it, better to keep the code without unchecked, it will just cost 3, 4 cents more to deploy it on layer like polygon that's ok

Reply:
Sure, I am very likely to use some of the code for my own projects, copying code without any improvement is haram in view